zomes/hc_cz_profile/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hc_cz_profile"
-version = "0.4.0-dev.3"
+version = "0.4.0-dev.7"
```

Comment:
This version should be the same as the `hdk` version

Reply:
as opposed to the holochain version? i thought it should match the holochain version